### PR TITLE
fix quick start code and doc

### PR DESCRIPTION
**Pull request by littletomatodonkey**
as title.

1. add lr_multi config in resnet_vd
2. fix jsdiv loss calculation
3. fix quick start doc
4. add model link about res2net ssld models

### CODE REVIEW

File: docs/zh_CN/tutorials/quick_start.md [hunk 1]
```diff
@@ -40,25 +40,23 @@ cd ../../
...
+tar -xf ResNet50_vd_ssld_pretrained.tar
+tar -xf MobileNetV3_large_x1_0_pretrained.tar
+cd ../
 ```
 
-参数说明：
-+ `architecture`（简写 a）：模型结构
-+ `path`（简写 p）：下载路径
-+ `decompress` （简写 d）：是否解压
+**注意**：如果是在windows中下载预训练模型的话，需要把地址拷贝到网页中并下载，同时使用`7Zip`等工具进行解压。
```
Comment (TingquanGao) [inline docs/zh_CN/tutorials/quick_start.md:59]: 需要将地址拷贝到浏览器中并下载，然后使用`7Zip`等工具进行解压。

File: docs/en/tutorials/quick_start_en.md [hunk 2]
```diff
@@ -40,33 +40,28 @@ cd ../../
...
+tar -xf MobileNetV3_large_x1_0_pretrained.tar
+cd ../
 ```
 
-Paramters：
-+ `architecture`(shortname: a): model name.
-+ `path`(shortname: p) download path.
-+ `decompress`(shortname: d) whether to decompress.
-
+**Note**: If you want to download the pretrained models on Windows environment, you can copy the links to the web page and download, you can use the the thirdparty tools such as `7Zip` to uncompress the tar files.
```
Comment (TingquanGao) [inline docs/en/tutorials/quick_start_en.md:59]: you can copy the links to the browser and download, then use the the thirdparty tools such as `7Zip` to uncompress the tar files.

**Review by TingquanGao — COMMENTED** [review]
几处用词似乎不太严谨，详见评论。

File: docs/zh_CN/tutorials/quick_start.md (hunk 1 above)
Comment (littletomatodonkey) [reply docs/zh_CN/tutorials/quick_start.md]: done, thx

File: docs/en/tutorials/quick_start_en.md (hunk 2 above)
Comment (littletomatodonkey) [reply docs/en/tutorials/quick_start_en.md]: done, thx

**Pull request merged**